ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -239,6 +240,31 @@ const moduleLabels: Map<AccessModule, JSX.Element> = new Map([
   [AccessModule.DATAUSERCODEOFCONDUCT, <div>Sign Data User Code of Conduct</div>],
 ]);
 
+const withAborter = async(abortableFn: (AbortController) => void) => {
+  const aborter = new AbortController();
+  try {
+    await abortableFn(aborter);
+  } catch (e) {
+    if (!isAbortError(e)) { throw e; }
+  } finally {
+    aborter.abort();
+  }
+};
+
+const moduleRefreshActions: Map<AccessModule, Function> = new Map([
+  [AccessModule.TWOFACTORAUTH, async() => {
+    await withAborter(aborter => profileApi().syncTwoFactorAuthStatus({signal: aborter.signal}));
+  }],
+  [AccessModule.RASLINKLOGINGOV, () => redirectToRas(false)],
```

Comment:
I think it make sense to not open new tab, and just use the existing RW page. In a happy case, this page becomes login.gov, then after user finish things again, it becomes RW site again.
But I am not sure if anything would went wrong, and make sure stuck in RAS page. But if that is rare, I think it's fine if user just open a new tab and go to RW again themself.

Reply:
Note that this is only for the refresh case.  For the normal "Verify your identity" step, it will still open a new tab.